backend/app/core/conf.py

```diff
@@ -45,6 +45,10 @@ class Settings(BaseSettings):
     # Env Opera Log
     OPERA_LOG_ENCRYPT_SECRET_KEY: str  # 密钥 os.urandom(32), 需使用 bytes.hex() 方法转换为 str
 
+    # OAuth2
+    OAUTH2_GITHUB_CLIENT_ID: str
```

Comment:
Whether to add the corresponding source link

Reply:
Good idea.